java/example_code/cloudmap/src/main/java/aws/example/cloudmap/DiscoverInstances.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Author : Prakash.Ramesh
+ */
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// This file is licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License. A copy of
+// the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+// snippet-sourcedescription:[DiscoverInstanceRequest.java demonstrates how to ....]
```

Comment:
The name of the file you are submitting is actually DiscoverInstances.java not DiscoverInstanceRequest.java

Can you please update it and finish the sentence for what the code example does.

Reply:
Fixed the file name and committed back. Please let me know for any other changes.

Thanks,
Prakash